config/application.rb

```diff
@@ -14,11 +14,11 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    config.time_zone = 'Moscow'
 
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
+    config.i18n.default_locale = :en
```

Comment:
With locale :ru some test is fails :(
<code>
$ rspec spec/models/answer_spec.rb

Answer
  should require body to be set (FAILED - 1)
  should ensure body has a length of at least 10 (FAILED - 2)
  should belong to question

Failures:

  1) Answer should require body to be set
     Failure/Error: it { should validate_presence_of :body }

     I18n::InvalidLocale:
       :ru is not a valid locale
     # ./spec/models/answer_spec.rb:5:in `block (2 levels) in <top (required)>'

  2) Answer should ensure body has a length of at least 10
     Failure/Error: it { should validate_length_of(:body).is_at_least(10) }

     I18n::InvalidLocale:
       :ru is not a valid locale
     # ./spec/models/answer_spec.rb:6:in `block (2 levels) in <top (required)>'

Finished in 0.038 seconds (files took 9.72 seconds to load)
3 examples, 2 failures

Failed examples:

rspec ./spec/models/answer_spec.rb:5 # Answer should require body to be set
rspec ./spec/models/answer_spec.rb:6 # Answer should ensure body has a length of at least 10
</code>